src/networkmessage.cpp

```diff
@@ -154,7 +154,7 @@ void NetworkMessage::addItem(const Item* item)
 
 	// display outfit on the podium
 	if (it.isPodium()) {
-		const Podium* podium = item->getPodium();
+		auto podium = item->getPodium();
```

Comment:
Could this be `const auto`
```suggestion
		const auto podium = item->getPodium();
```